tutorials/nservicebus-101/lesson-2/Snippets/Core_6/ComplexCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using NServiceBus;
+
+namespace Core_6
+{
+    #region ComplexCommand
+
+    public class DoSomethingComplex :
+        ICommand
+    {
+        public int SomeId { get; set; }
+        public ChildClass ChildStuff { get; set; }
```

Comment:
Should we encourage people to use complex contracts? I think the guidance should be to keep the message structure as flat as possible.